src/models/CommonSchema.ts

```diff
@@ -24,7 +24,8 @@ export const testTypesCommonSchema = {
     prohibitionIssued: Joi.boolean().required().allow(null),
     reasonForAbandoning: Joi.string().required().allow("", null),
     additionalNotesRecorded: Joi.string().max(500).required().allow("", null),
-    additionalCommentsForAbandon: Joi.string().max(500).required().allow("", null)
+    additionalCommentsForAbandon: Joi.string().max(500).required().allow("", null),
+    testExpiryDate: Joi.date().iso().allow(null),
```

Comment:
@KHodderDVSA I moved it here

Reply:
Based on your other comment, this should not allow  null, yeah?